golang/cosmos/types/kv_entry_helpers.go

```diff
@@ -232,3 +232,45 @@ func EncodeKVEntryReaderToJsonl(reader KVEntryReader, bytesWriter io.Writer) (er
 		}
 	}
 }
+
+var _ KVEntryReader = &kvHookingReader{}
```

Comment:
_fix_:

Remove this line. I can't see why this is necessary and compiling without it succeeds.

Reply:
This is an interface check, making sure `kvHookingReader` implements `KVEntryReader`. There's lots of those around, and I simply cargo culted, but open to alternatives.

Reply:
I prefer the more obviously useless:
```suggestion
var _ KVEntryReader = (*kvHookingReader)(nil)
```

Reply:
Oh gosh... I completely forgot the use of these in golang. Yeah, happy for this to stay.